proof_of_stake/src/lib.rs

```diff
@@ -250,6 +250,12 @@ pub fn below_capacity_validator_set_handle() -> BelowCapacityValidatorSets {
     BelowCapacityValidatorSets::open(key)
 }
 
+/// Get the storage handle to the epoched below-threshold validator set
+pub fn below_threshold_validator_set_handle() -> BelowThresholdValidatorSets {
```

Comment:
I think we can avoid storing the 3rd set altogether as we don't need to keep an order for these validators. When the validator state is `BelowThreshold` it should be in neither consensus or below_capacity set. This will reduce storage cost and we won't need to copy these on a new epoch.